layouts/partials/header.html

```diff
@@ -17,12 +17,25 @@
     <link rel="icon" type="image/png" href="{{ "favicon-16x16.png" | relURL }}" sizes="16x16">
     <link rel="icon" type="image/png" href="{{ "favicon-128.png" | relURL }}" sizes="128x128">
     <meta name="application-name" content="Apache Camel">
-    <meta property="og:title" content="Apache Camel helps you integrate over 300 different systems!">
-    <meta property="og:site_name" content="Apache Camel">
-    <meta property="og:url" content="https://camel.apache.org">
-    <meta property="og:description" content="Camel is an open source integration framework that empowers you to quickly and easily integrate various systems consuming or producing data.">
+    
+    <meta property="og:title" content="{{ .Title }}"> 
+    <meta property="og:site_name" content="{{ .Site.Title }}"> 
+    <meta property="og:url" content="{{ .Permalink }}"> 
+    {{ if .Description}}
+        <meta name="og:description" content="{{ .Description }}">
```

Comment:
Where does the `.Description` come from? I can see only that some security announcements have a `description` front matter set.

Reply:
No idea. I am reusing the same definition for meta-description: https://github.com/apache/camel-website/pull/767/files#diff-42986cf40ad96f5b7ca39d9b71ba6a9f252656de24cfe82100892ff0f4400f90L29 - if it is presents, then it should be used, otherwise we may fallback to other variables, such as the preview that we use in the blog.